src/Styles/StyledComponents.js

```diff
@@ -74,13 +140,13 @@ export const List = styled.ul`
   list-style: none;
   padding-top: 2em;
 
-  @media (min-width: 375px) and (max-width: 480px) {
+  @media (min-width: 320px) and (max-width: 480px) {
```

Comment:
having all of the styles in a small app like this is probably fine, but it could quickly become hard to maintain.

Reply:
I will do my best to try to make the components reusable throughout the whole site.